pkg/registry/registry_antrea.csv

```diff
@@ -36,4 +36,8 @@ ElementID,Name,Abstract Data Type,Data Type Semantics,Status,Description,Units,R
 134,reversePacketDeltaCountFromDestinationNode,unsigned64,,current,,,,,,,,56506,
 135,reverseOctetDeltaCountFromDestinationNode,unsigned64,,current,,,,,,,,56506,
 136,tcpState,string,,current,,,,,,,,56506,
-137,flowType,unsigned8,,current,,,,,,,,56506,
+137,flowType,unsigned8,,current,Type of flow based on location of source and destination. 1-IntraNode 2-InterNode 3-ToExternal 4-FromExternal,,,,,,,56506,
+138,tcpStatePrev,string,,current,,,,,,,,56506,
+139,tcpStatePrevList,string,,current,,,,,,,,56506,
```

Comment:
why both? If there is only one state, it can still be part of the list.

Reply:
Removed `tcpStatePrev` since we can keep states in `tcpStatePrevList` in ordered list.